### PR TITLE
Update use-drive-shipping-to-import-pst-files-to-office-365.md

### PR DESCRIPTION
**Pull request by Durga-SC**
removed storage key @markjjo

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/use-drive-shipping-to-import-pst-files-to-office-365.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Here are the steps required to use drive shipping to import PST files to Microso
 [Step 6: Filter data and start the PST Import job](#step-6-filter-data-and-start-the-pst-import-job)
   
 > [!IMPORTANT]
-> You have to perform Step 1 once to down load the secure storage key and the import tool. After you perform these steps, follow Step 2 through Step 6 each time you want to ship a hard drive to Microsoft. 
+> You have to perform Step 1 once to down load the import tool. After you perform these steps, follow Step 2 through Step 6 each time you want to ship a hard drive to Microsoft.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-drive-shipping-to-import-pst-files-to-office-365.md]:
```suggestion
> You have to perform Step 1 once to download the import tool. After you have performed these steps, follow Step 2 through Step 6 each time you want to ship a hard drive to Microsoft. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Durga-SC, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Pull request merged**